otcextensions/sdk/vpcep/v1/endpoint.py

```diff
@@ -0,0 +1,92 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+from openstack import resource
+
+
+class TagsSpec(resource.Resource):
+    #: Specifies the tag key.
+    key = resource.Body('key')
+    #: Specifies the tag value.
+    value = resource.Body('value')
+
+
+class Endpoint(resource.Resource):
+    base_path = '/vpc-endpoints'
+    resources_key = 'endpoints'
+    resource_key = None
+
+    # capabilities
+    allow_create = True
+    allow_fetch = True
+    allow_delete = True
+    allow_list = True
+
+    _query_mapping = resource.QueryParameters(
+        'id',
+        'offset',
+        'limit',
+        'router_id',
+        'endpoint_service_name',
+        'sort_dir',
+        'sort_key',
+        router_id='vpc_id',
+    )
+
+    # Properties
+    #: Domain status.
+    active_status = resource.Body('active_status', type=list)
+    #: Creation time of the VPC endpoint.
+    created_at = resource.Body('created_at')
+    #: Description of the VPC endpoint.
+    description = resource.Body('description')
+    #: Domain name for accessing the associated VPC endpoint service.
+    dns_names = resource.Body('dns_names', type=list)
+    #: Whether the VPC endpoint is enabled.
+    enable_status = resource.Body('enable_status')
```

Comment:
is_enabled

Reply:
Hi @anton-sidelnikov , its string variable  ,"enable_status":"enable",  thats why i didnt change the name to is_enabled

Reply:
updated `enable_status` to `is_enabled` (type=bool)